actions/parse-push/__tests__/main.test.ts

```diff
@@ -26,40 +26,52 @@
 import nock from 'nock'
 import path from 'path'
 
-import { getPullRequestHtmlUrl } from '../src/main'
+import { getPullRequestHtmlUrl } from '../src/run'
 
 const fixtures = path.join(__dirname, 'fixtures')
 
 beforeAll(() => {
-    // Disable executing of runner commands when running tests in actions
-    console.log('::stop-commands::stoptoken')
+    nock.disableNetConnect()
 })
 
 afterAll(() => {
-    // Re-enable executing of runner commands when running tests in actions
-    console.log('::stoptoken::')
+    nock.cleanAll()
+    nock.enableNetConnect()
 })
 
 describe('commit construction functions', () => {
     describe('pull_request_html_url', () => {
-        it('correctly extracts the HTML URL from a commit with a single associated PR', () => {
+        it('correctly extracts the HTML URL from a commit with a single associated PR', async () => {
             const commit = '1ec45d4ca4ccae5c3a68c2cf319d29c8ab4028e2'
+
             nock('https://api.github.com')
                 .persist()
```

Comment:
I think we don't actually want this since we only want to mock one call

Reply:
You do want this. You mock a "context" (aka domain + tld), and by default it will blow up if you're doing extra/unmocked calls. So you're safe!